src/geode/mesh/core/edged_curve.cpp

```diff
@@ -151,7 +142,7 @@ namespace geode
         BoundingBox< dimension > box;
         for( const auto p : Range{ nb_vertices() } )
         {
-            box.add_point( point( p ) );
+            box.add_point( this->point( p ) );

```

Comment:
Should we define a method `point` in CRSManagers inherited classes to avoid adding `this->` at each call?